specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

```diff
@@ -0,0 +1,949 @@
+        "description": "Collection ids are user-created collections of ledger entries",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of collections",
+            "schema": {
+              "$ref": "#/definitions/CollectionArray"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      }
+    },
+    "/app/transactions": {
+      "get": {
+        "operationId": "ConfidentialLedger_GetLedgerEntries",
+        "x-ms-examples": {
+          "GetLedgerEntries": {
+            "$ref": "./examples/ledger/GetLedgerEntries.json"
+          }
+        },
+        "x-ms-pageable": {
+          "itemName": "entries",
+          "nextLinkName": "nextLink"
+        },
+        "summary": "Gets ledger entries from a collection corresponding to a range.",
+        "description": "A collection id may optionally be specified. Only entries in the specified (or default) collection will be returned.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CollectionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/TransactionIdFromParameter"
+          },
+          {
+            "$ref": "#/parameters/TransactionIdToParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ledger entries in the specified range.",
+            "schema": {
+              "$ref": "#/definitions/PagedLedgerEntries"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      },
+      "post": {
+        "operationId": "ConfidentialLedger_PostLedgerEntry",
+        "x-ms-examples": {
+          "PostLedgerEntry": {
+            "$ref": "./examples/ledger/PostLedgerEntry.json"
+          }
+        },
+        "summary": "Writes a ledger entry.",
+        "description": "A collection id may optionally be specified.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CollectionIdParameter"
+          },
+          {
+            "in": "body",
+            "name": "entry",
+            "description": "Ledger entry.",
+            "schema": {
+              "$ref": "#/definitions/LedgerEntry"
+            }
```

Comment:
This body parameter is not marked as required, but the schema has required properties. This is likely an error.